pyproject.toml

```diff
@@ -91,3 +90,17 @@ line-length = 120
 # Configure hatch for python builds
 [tool.hatch]
 version.source = "vcs"
+build.include = [
+    "generative_data_prep/**",
+    "requirements/**",
```

Comment:
Do we have proper control at `.gitignore` to prevent accidental check-ins? I assume everything under here will go into the package, but not sure if there could be auto-generated files here.

Reply:
yeah, it respects all the ignored files in `.gitignore` if that's what you're asking